src/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+class ToDoList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            input: '',
+            toDoItems: []
+        }
+    }
+
+    addItem = () => {
+        if (this.state.input !== '') {
+            this.setState(prevState => ({
+                toDoItems: [...prevState.toDoItems, this.state.input]
+            }));
+        }
```

Comment:
setState can also be written as "this.setState({ toDoItems: [...this.state.toDoItems, this.state.input] })". Not that you're function is wrong, I cannot remember the purpose of doing it your way, so I prefer this :)